src/style.css

```diff
@@ -0,0 +1,109 @@
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 11px;
+}
+
+.block {
+  display: flex;
+}
+
+.card {
+  flex: 200px 0 0;
+  justify-content: center;
+  padding: 15px 14px;
+  background: #fff;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  display: block;
+  margin: 15px auto 45px;
+}
+
+.card__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+}
+
+.card__code {
+  line-height: 13px;
+  margin-top: 5px;
+  margin-bottom: 15px;
+}
+
+.card__info {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 20px;
+}
+
+.card__info--price {
+  align-items: flex-end;
+}
+
+.card__stars {
+  display: flex;
+}
+
+.card__star {
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+}
+
+.card__stars--4 .card__star:nth-child(-n+4) {
+  background-image: url("images/star-active.svg");
+}
+
+.card__text {
+  line-height: 13px;
+}
+
+.card__text--price {
+  margin-bottom: 1px;
+}
+
+.card__number {
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 19px;
+}
+
+.card__btn {
+  width: 170px;
+  height: 40px;
+  background: #00acdc;
+  border-radius: 5px;
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 16px;
+  text-align: center;
+  text-transform: uppercase;
+  box-sizing: border-box;
+}
+
+.card__btn-text {
+  display: block;
+  line-height: 40px;
+  color: #fff;
+  text-decoration: none;
+}
+
+.card__btn:hover {
+  background-color: #fff;
+  border: solid 1px #00acdc;
```

Comment:
at this moment when you hovered on your link - border appears and adds 1px, because of box-sizing: border-box; width and height of your button is recalculating, and text could change his position, so its better if you write border property inside the button